cli/src/wallet.rs

```diff
@@ -1,88 +1,105 @@
+                return Ok((None, None));
+            }
+            WalletCommand::ChangePassword => {
+                let wallet = change_password_command(storage_path, snapshot_path).await?;
+                (Some(wallet), None)
+            }
+            WalletCommand::New { alias } => {
+                let (wallet, account) = new_command(storage_path, snapshot_path, alias).await?;
+                (Some(wallet), Some(account))
+            }
+            WalletCommand::SetNode { url } => {
+                let wallet = set_node_command(storage_path, snapshot_path, url).await?;
+                (Some(wallet), None)
+            }
+            WalletCommand::Sync => {
+                let wallet = sync_command(storage_path, snapshot_path).await?;
+                (Some(wallet), None)
+            }
+            WalletCommand::Mnemonic => {
+                mnemonic_command().await?;
+                return Ok((None, None));
+            }
+        }
+    } else {
+        // no command provided, i.e. `> ./wallet`
+        match (storage_path.exists(), snapshot_path.exists()) {
+            (true, true) => {
+                let password = get_password("Stronghold password", false)?;
+                let wallet = unlock_wallet(storage_path, snapshot_path, &password).await?;
+                if wallet.get_accounts().await?.is_empty() {
+                    // ask the user whether a default account should be created
+                    if get_decision("Initialize a default account?")? {
+                        let account = add_account(&wallet, None).await?;
+                        println_log_info!("Initialized default account.");
+                        (Some(wallet), Some(account))
+                    } else {
+                        (Some(wallet), None)
+                    }
+                } else if let Some(account_handle) = pick_account(&wallet).await? {
+                    let account = account_handle.alias().await;
+                    (Some(wallet), Some(account))
+                } else {
+                    (Some(wallet), None)
                 }
-                WalletCommand::ChangePassword => change_password_command(&wallet, &password).await?,
-                WalletCommand::New { alias } => account = Some(new_command(&wallet, alias).await?),
-                WalletCommand::SetNode { url } => set_node_command(&wallet, url).await?,
-                WalletCommand::Sync => sync_command(&wallet).await?,
-                // PANIC: this will never happen because these variants have already been checked.
-                WalletCommand::Init(_) | WalletCommand::Mnemonic | WalletCommand::Restore { .. } => unreachable!(),
-            };
+            }
+            (false, false) => {
+                if get_decision("Initialize a new wallet with default values?")? {
+                    let wallet = init_command(storage_path, snapshot_path, InitParameters::default()).await?;
```

Comment:
This command prompts the user to store the mnemonic without telling them what is happening. It is a bit confusing:

```
Initialize a new wallet with default values?: y
Stronghold password: [hidden]
Mnemonic has been generated.
❯ Write to console only
  Write to file only
  Write to console and file
```

It should perhaps say: `Where do you want to store the generated mnemonic?`

Reply:
I have no problem with changing it, but the term "store" would be confusing here IMO because in the text that comes afterwards we are requesting the user to store it. So it's really just written either to the console or to a file, which then should be stored by the user, either by copying it to a secure location from the console, or by moving the created "mnemonic.txt" file to a secure/encrypted device.

Reply:
I think it would already be a lot clearer if we say "Write it to..." then it's obvious we're speaking about the mnemonic

Reply:
right ... one small word can make all the difference